README.md

```diff
@@ -122,6 +122,17 @@ python_version = 3.12
 python_full_version = 3.12.7
 implementation_name = cpython
 
+[env.windows]
+os_name = nt
+platform_tag = win_amd64
+sys_platform = (win|win32)
```

Comment:
I copied this from #26, but I'm unsure if win32 is needed, thoughts?

Reply:
Thanks.

Reply:
I have done some testing now, and at least with the packages I used it made no difference whether I used win or win32, so I guess it would be best to go with the one used in the documentation.